code/src/reducers/game.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable linebreak-style */
+import { createSlice } from '@reduxjs/toolkit';
+import { ApiStart, ApiMove } from '../utils/api.js';
+
+export const game = createSlice({
+  name: 'game',
+  initialState: {
+    username: null,
+    position: null
+  },
+  reducers: {
+    addPlayer: (store, action) => {
+      store.username = `${new Date().getTime()} ${action.payload}`
```

Comment:
nice way to get somewhat unique username 👍

Reply:
Why not use "uniqid" on the username as well? This would make the reducer more readable. But hey it works 😋